doc/examples/config/replicaset.yaml

```diff
@@ -1,12 +1,10 @@
 credentials:
```

Comment:
Regarding the first patch and its NO_DOC/NO_CHANGELOG statements.

> NO_DOC=removed feature was not released yet
> NO_CHANGELOG=removed feature was not released yet

There is a pre-release 3.0.0-alpha1. We already have a documentation issue (https://github.com/tarantool/doc/issues/3544), but it is not very detailed and usually remains actual after changes like the given one. I like how it is described in your commit 229044231e0f8ec94b78fb442f05027399378abc and would appreciate if you'll do the same here.

I want a list of changes from 3.0.0-alpha1 in the upcoming pre-release 3.0.0-alpha2, so it would be good to have a brief changelog entry together with the patch.

Reply:
Done:
```md
  ## feature/config
  
  * Removed sha1 and sha256 hash entries from config credentials schema,
    as not planned for development (gh-8967).
```

Reply:
Updated NO_DOC too.